src/scaffold/web_app/uis.rs

```diff
@@ -90,10 +109,11 @@ impl TryFrom<&FileTree> for UiFramework {
 impl std::fmt::Display for UiFramework {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let str = match self {
-            UiFramework::Vanilla => "vanilla",
-            UiFramework::Lit => "lit",
-            UiFramework::Svelte => "svelte",
-            UiFramework::Vue => "vue",
+            UiFramework::Vanilla => "vanilla".yellow(),
+            UiFramework::Lit => "lit".blue(),
```

Comment:
The blue is hardly visible in my Ubuntu terminal:
![image](https://github.com/holochain/scaffolding/assets/36768177/b30cf63b-8c8a-43db-9566-2ffa94290126)

I wonder whether `bright_blue` might be better.

Reply:
Fixed